packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,173 @@
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import React from 'react';
+import { designTokenStory } from './util';
+
+const defaultArgs = {
+  dir: 'auto',
+  disabled: false,
+  focus: false,
+  hover: false,
+  invalid: false,
+  placeholder: undefined,
+  readOnly: false,
+  required: false,
+  spellCheck: false,
+  children: undefined,
+};
+
+const exampleArgs = {
+  children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+};
+
+const arabicDecorator = (Story) => (
+  <div dir="rtl" lang="ar">
+    {Story()}
+  </div>
+);
+
+const meta = {
+  title: 'CSS Component/Textarea',
+  id: 'css-textarea',
+  component: Textarea,
+  argTypes: {
+    disabled: {
+      description: 'Disabled',
+      control: 'boolean',
+    },
+    invalid: {
+      description: 'Invalid',
+      control: 'boolean',
+    },
+    readOnly: {
+      description: 'Read-only',
+      control: 'boolean',
+    },
+    required: {
+      description: 'Required',
+      control: 'boolean',
+    },
+  },
+  args: {},
+  tags: ['autodocs'],
+  parameters: {
+    status: {
+      type: 'WORK IN PROGRESS',
+    },
+    tokensPrefix: 'utrecht-textarea',
+    tokens,
+    tokensDefinition,
+    docs: {
+      description: {
+        story: `Styling via the \`.utrecht-textarea\` class name.`,
+        component: readme,
+      },
+    },
+  },
+} satisfies Meta<typeof Textarea>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    ...defaultArgs,
+  },
+};
+
+export const DefaultValue: Story = {
+  args: {
+    ...defaultArgs,
+  },
+  render: () => {
+    return <Textarea defaultValue={exampleArgs.children} />;
+  },
+};
+
+export const Direction: Story = {
+  args: {
+    ...defaultArgs,
+  },
+  decorators: [arabicDecorator],
+};
+
+export const Placeholder: Story = {
+  name: 'Placeholder',
+  args: {
+    ...defaultArgs,
+    placeholder: 'Typ je bericht...',
+  },
+};
+
+export const PlaceholderLeftToRight: Story = {
+  name: 'Placeholder (right-to-left)',
+  args: {
+    ...defaultArgs,
+    placeholder: 'اكتب رسالتك',
+  },
+  decorators: [arabicDecorator],
+};
+
+export const Spellcheck: Story = {
+  args: {
+    ...defaultArgs,
+  },
+  argTypes: {
+    ...Default.argTypes,
+    spellCheck: {
+      description: 'Spellcheck',
+      control: { type: 'select' },
+      options: ['', 'false', 'true'],
+    },
+  },
+};
+
+export const Disabled: Story = {
+  args: {
+    ...Default.args,
+    disabled: true,
+  },
+};
+
+export const Invalid: Story = {
+  args: {
+    ...Default.args,
+    invalid: true,
+  },
+};
+
+export const ReadOnly: Story = {
+  args: {
+    ...Default.args,
+    readOnly: true,
+  },
+};
+
+export const Required: Story = {
+  args: {
+    ...Default.args,
+    required: true,
+  },
+};
+
+export const Focus: Story = {
+  args: {
+    ...Default.args,
+    focus: true,
+  },
+};
+
+export const Hover: Story = {
+  args: {
+    ...Default.args,
+    hover: true,
+  },
```

Comment:
Hover simulation is broken now.

## Main branch

```
<textarea class="utrecht-textarea utrecht-textarea--html-textarea utrecht-textarea--hover">
  The Quick Brown Fox Jumps Over The Lazy Dog
</textarea>
```

## This branch

```
<textarea
  spellcheck="false"
  class="utrecht-textarea utrecht-textarea--html-textarea"
  dir="auto"
></textarea>
```

Class name is missing.